src/pg/connection.cr

```diff
@@ -2,6 +2,7 @@ require "../pq/*"
 
 module PG
   class Connection < ::DB::Connection
+    property time_zone = Time::Location.local
```

Comment:
I used local time here, but I figure for backwards compatibility we'd probably want it to be UTC?